nan_implementation_pre_12_inl.h

```diff
@@ -185,7 +186,8 @@ Factory<v8::String>::New(const uint8_t * value, int length) {
   if (wideString.size() == 0) {
     return v8::String::Empty();
   } else {
-    return v8::String::New(&wideString.front(), wideString.size());
+    return v8::String::New(&wideString.front()
+         , static_cast<int>(wideString.size()));
```

Comment:
Moved the assert in widenString outside the branch instead. Now `wideString.size()` is guaranteed to be `<= INT_MAX` at this point.